tools/advisor/advisor/db_stats_fetcher.py

```diff
@@ -1,16 +1,81 @@
-from advisor.db_timeseries_parser import TimeSeriesData
+from advisor.db_log_parser import Log
+from advisor.db_timeseries_parser import TimeSeriesData, NO_ENTITY
+import glob
+import re
 import subprocess
+import time
 
 
-class OdsTimeSeriesData(TimeSeriesData):
+class DBBenchStatsParser(TimeSeriesData):
```

Comment:
This would be LogStatesParser. DBBench is just a program that runs the DB and that stats that we want to parse will be located in the LOG file of the db.